### DISCUSSION TITLE
Wallet Connect v2 Sign API Problem and Goal

### DISCUSSION THREAD

**Discussion by Luzzotica** [WalletConnect/walletconnect-monorepo#1710] [Developer Support]
I see that the Sign API for wallet connect v2 states a pretty significant problem regarding wallet connect v2.

I'm curious what the discussion surrounding this is, what the current options are, and if a conclusion has already been made and the docs just haven't been updated.

I'm looking into creating a dart (flutter) SDK and I'd like to know if this problem has been resolved.

**Comment by chadyj** [comment]
Hey @Luzzotica what is the problem you are referring to?

**Reply by Luzzotica** [reply]
@chadyj 
https://docs.walletconnect.com/2.0/specs/clients/sign/

The problem and goal listed here. Has there been any update on this? Are the docs just out of date?

**Reply by chadyj** [reply]
Sign v2 has launched. 
This was the problem definition in the specs.

Try it out here https://docs.walletconnect.com/2.0/api/sign
